wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -1968,6 +1970,26 @@ class KotlinGenerator private constructor(
     }
   }
 
+  private fun addBuildClosure(type: MessageType, companionBuilder: TypeSpec.Builder) {
+    val body = buildCodeBlock {
+      addStatement("val builder = Builder()")
+      addStatement("buildFn(builder)")
+      addStatement("return builder.build()")
+    }
+    val buildFn = FunSpec.builder("build")
+      .addParameter("buildFn",
```

Comment:
Generally we avoid abbreviations in names. I think "builder" or "body" would work here.